src/test/components/Root.test.js

```diff
@@ -72,33 +113,41 @@ describe('app/ProfileViewWhenReady', function() {
         })
       )
     );
-    expect(view.dive()).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   it('renders an home when the user pressed back after an error', function() {
     const {
-      view,
+      container,
       dispatch,
       navigateToStoreLoadingPage,
       navigateBackToHome,
     } = setup();
 
     navigateToStoreLoadingPage();
     dispatch(fatalError(new Error('Error while loading profile')));
-    expect(view.dive()).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
     expect(console.error).toHaveBeenCalled();
 
     navigateBackToHome();
-    expect(view.dive()).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
   });
 });
 
 function setup() {
   // Let's silence the error output to the console
   jest.spyOn(console, 'error').mockImplementation(() => {});
+  // $FlowFixMe Flow doesn't know retrieveProfileFromAddon is a jest mock.
```

Comment:
We can remove these `$FlowFixMe`s and change the following lines with wrapping like `(<something> : any)`.
It's not elegant but it solves the problem at least.

```suggestion
  // Flow doesn't know retrieveProfileFromAddon is a jest mock.
```